includes/KSES.php

```diff
@@ -525,21 +535,61 @@ public function filter_kses_allowed_html( $allowed_tags ) {
 				'srcset'        => true,
 				'srcwidth'      => true,
 			],
+			'svg'                       => [
```

Comment:
SVG has more attributes - https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg

Reply:
True, but at the moment we‘re only using a limited set of attributes for our shapes / masks.

Reply:
See https://github.com/google/web-stories-wp/blob/5e3fdb9078e0836e6eeaf3cf36149147a9d2b5eb/assets/src/edit-story/masks/output.js#L95-L105